modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -0,0 +1,216 @@
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker <1>
+  name: 10-br-ex-worker <2>
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,<base64_encoded_nmstate_configuration> <3>
+        mode: 0644
+        overwrite: true
+        path: /etc/nmstate/openshift/cluster.yml
+# ...
+----
+<1> For each node in your cluster, specify the hostname path to your node and the base-64 encoded Ignition configuration file data for the machine type. If you have a single global configuration specified in an `/etc/nmstate/openshift/cluster.yml` configuration file that you want to apply to all nodes in your cluster, you do not need to specify the hostname path for each node. The `worker` role is the default role for nodes in your cluster. Note that the `.yaml` extension does not work when specifying the hostname path for each node or all nodes in the `MachineConfig` manifest file.
+<2> The name of the policy.
+<3> Writes the encoded base64 information to the specified path. 
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+ifdef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+. Create a `NodeNetworkConfigurationPolicy (NNCP)` CR and define a customized `br-ex` bridge network configuration. Depending on your needs, ensure that you set a masquerade IP for either the `ipv4.address.ip`, `ipv6.address.ip`, or both parameters. A masquerade IP address must match an in-use IP address block.
++
+.Example of an NNCP CR that sets IPv6 and IPv4 masquerade IP addresses.
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: worker-0-br-ex <1>
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: worker-0
+    desiredState:
+    interfaces:
+    - name: enp2s0 <2>
+      type: ethernet <3>
+      state: up <4>
+      ipv4:
+        enabled: false <5>
+      ipv6:
+        enabled: false
+    - name: br-ex
+      type: ovs-bridge
+      state: up
+      ipv4:
+        enabled: false
+        dhcp: false
+      ipv6:
+        enabled: false
+        dhcp: false
+      bridge:
+        port:
+        - name: enp2s0 <6>
+        - name: br-ex
+    - name: br-ex
+      type: ovs-interface
+      state: up
+      copy-mac-from: enp2s0
+      ipv4:
+        enabled: true
+        dhcp: true
+        address:
+        - ip: "169.254.169.2"
+          prefix-length: 29
+      ipv6:
+        enabled: false
+        dhcp: false
+        address:
+        - ip: "fd69::2"
+        prefix-length: 125
+----
+<1> Name of the policy.
+<2> Name of the interface.
+<3> The type of interface. This example creates a bridge.
+<4> The requested state for the interface after creation.
+<5> Disables IPv4 and IPv6 in this example.
+<6> The node NIC to which the bridge is attached.
+
+. Reboot your cluster for the `br-ex` bridge changes to take effect.
```

Comment:
If using NNCP to update the br-ex bridge configuration via day2 operator(kubernetes-nmstate operator), there is no need to reboot the cluster to make the changes to take effect.

Reply:
Thanks. Removed, but added reboot step to Day 0 if that is correct?

Reply:
No need to reboot at Day0 either.

Reply:
Thanks, updated.